Enable drjit python for mac-arm64

jeongseok-meta opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19715121271](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19715121271). Examine the logs at this URL for more detail.</sub>

conda-forge-admin commented:
Hi! This is the friendly conda-forge automerge bot!

I considered the following status checks when analyzing this PR:
 - **linter**: passed
 - **azure**: failed


Thus the PR was not passing and not merged.

conda-forge-admin commented:
Hi! This is the friendly conda-forge automerge bot!

Commits were made to this PR after the `automerge` label was added. For security reasons, I have disabled automerge by removing the `automerge` label. Please add the `automerge` label again (or ask a maintainer to do so) if you'd like to enable automerge again!

conda-forge-admin commented:
Hi! This is the friendly conda-forge automerge bot!

Commits were made to this PR after the `automerge` label was added. For security reasons, I have disabled automerge by removing the `automerge` label. Please add the `automerge` label again (or ask a maintainer to do so) if you'd like to enable automerge again!

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ The feedstock has no `.ci_support` files and thus will not build any packages.

For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19786733641](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19786733641). Examine the logs at this URL for more detail.</sub>

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19787254802](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19787254802). Examine the logs at this URL for more detail.</sub>

jeongseok-meta committed:
Enable macOS arm64 Python builds

jeongseok-meta committed:
MNT: Re-rendered with conda-smithy 3.61.2 and conda-forge-pinning 2026.05.09.13.09.17

Other tools:
- conda-build 26.3.0
- rattler-build 0.64.1
- rattler-build-conda-compat 1.4.14
